### PR TITLE
added count of holidays

### PR DESCRIPTION
**Pull request by Himanshi-Khandelwal**
modify holiday calender page

### CODE REVIEW

File: routes/users.js [hunk 1]
```diff
@@ -629,28 +629,10 @@ router.get('/dashboard2/:username/mytapplications', function(req, res){
 });
 
 router.get('/calendar', loggedIn , function(req, res){
-	res.render('calendar');
+	res.render('calendar',{username : req.params.username, h1 : req.user.holidays, h2 : req.user.halfdays , h3 : req.user.nonfunc_holidays, flag : req.user.flag });
```
Comment (sid22) [inline routes/users.js]: why are we sending username? Also, req.params is when we take parameters from url string, username isn't there in calendar url.

**Review by sid22 — COMMENTED** [review]
Remove, username: req.params.username, if you need username, make it as req.user.username.

**Comment by Himanshi-Khandelwal** [comment]
done

**Review by sid22 — APPROVED** [review]

**Comment by sid22** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/c74f8248-fea8-440b-bfd4-822ecb10c7be "emma-toolchain"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/0811a336-3049-4159-ba04-aaf43efd0023 "emma") deployed [leave](https://github.com/goyal-sidd/leave/releases/tag/deploy-ibm-yp-us-south-sid22-leave-20171102-122741 "deploy-ibm-yp-us-south-sid22-leave-20171102-122741") to [leave](https://console.bluemix.net/apps/0e816a54-7da8-4450-8e72-70dc537dc333?env_id=ibm:yp:us-south "ibm:yp:us-south:sid22:leave"), including commit 4d43ca3cf45a2675dca6171e6e1494fac444ae72

**Pull request merged**